libs/backendless.js

```diff
@@ -1332,6 +1332,34 @@
             argsObj.url = 'last';
 
             return this._find.apply(this, [argsObj].concat(Array.prototype.slice.call(arguments)));
+        },
+
+        /**
+         * Count of object
+         *
+         * @param {DataQueryBuilder} [dataQueryBuilder]
+         * @param {Async} [async]
+         *
+         * @return {*}
+         */
+        getObjectCount: function(queryBuilder, async) {
+            this._validateFindArguments(arguments);
+
+            var args = this._parseFindArguments(arguments);
```

Comment:
Wouldn't it be better if you move `_validateFindArguments` method call inside `_parseFindArguments` ?